src/helper/selection-tools/move-tool.js

```diff
@@ -101,7 +120,22 @@ class MoveTool {
         const point = event.point;
         point.x = Math.max(0, Math.min(point.x, ART_BOARD_WIDTH));
         point.y = Math.max(0, Math.min(point.y, ART_BOARD_HEIGHT));
+        
         const dragVector = point.subtract(event.downPoint);
+        let snapVector;
+
+        // Snapping to align center. Only in select mode, because only select shows a center
+        // crosshair to line up.
+        const center = new paper.Point(ART_BOARD_WIDTH / 2, ART_BOARD_HEIGHT / 2);
+        if (!event.modifiers.shift && this.mode === Modes.SELECT) {
```

Comment:
Doesn't this collide with shift+drag being used for straight/diagonal movement?

Reply:
This is if NOT shift, so doesn't change shift+drag behavior